components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.http/src/main/java/org/wso2/carbon/event/output/adapter/http/HTTPEventAdapter.java

```diff
@@ -98,11 +101,32 @@ public void init() throws OutputEventAdapterException {
             } else {
                 jobQueSize = HTTPEventAdapterConstants.ADAPTER_EXECUTOR_JOB_QUEUE_SIZE;
             }
-
             executorService = new ThreadPoolExecutor(minThread, maxThread, defaultKeepAliveTime, TimeUnit.MILLISECONDS,
                     new LinkedBlockingQueue<Runnable>(jobQueSize));
-        }
 
+            //configurations for the httpConnectionManager which will be shared by every http adapter
+            int defaultMaxConnectionsPerHost;
+            int maxTotalConnections;
+
+            if(globalProperties.get(HTTPEventAdapterConstants.DEFAULT_MAX_CONNECTIONS_PER_HOST) != null){
```

Comment:
Thanks @keizer619  - Done in pull request - https://github.com/wso2/carbon-analytics-common/pull/109